File: Data Structures & Algorithms/top-k-elements-in-list/submission-3.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        freq={}
        for el in nums:
            if el in freq:
                freq[el] += 1
            else:
                freq[el]=1
            ans=[]
        for key,value in sorted(freq.items(),key=lambda x:x[1],reverse=True):
            ans.append(key)
        return ans[0:k]

File: Data Structures & Algorithms/reverse-integer/submission-3.cpp
class Solution {
public:
    int reverse(int x) {
        int rev = 0;

        while (x != 0) {
            int R = x % 10;
            x = x / 10;

            if (rev > 214748364 || 
                (rev == 214748364 && R > 7)) {
                return 0;
            }

            if (rev < -214748364 || 
                (rev == -214748364 && R < -8)) {
                return 0;
            }

            rev = rev * 10 + R;
        }

        return rev;
    }
};